### PR TITLE
Add and fix a number of minor things.

### PR DESCRIPTION
**Pull request by beorn7**
- Sort exporters lexicographically (but still put the node exporter first).
- Call "unofficial" exporters "independently maintained" exporters and
  adjust the wording around it.
- Add the resurrected Munin exporter.
- Add the crucial information about RAM to the benchmark results.
- Add a new "media" section for introductory blog posts and talks.
- Refer to that section instead of the SC blog post in the Overview.

Note about Stapelberg's talk: As it is in German and without
subtitles, it is only approachable for a limited audience, but it is
in fact a very good introduction, so I opted to list it here in last
position. Measured in "number of Wikipedia articles", German is the
2nd most important language on the Internet after English. So I hope
it's not completely obscure.

@fabxc @juliusv @brian-brazil

### CODE REVIEW

File: content/docs/introduction/media.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+## Blogs
+
+* This site has its own [blog](http://prometheus.io/blog/).
+* [SoundCloud's blog post announcing Prometheus](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud) – a more elaborate overview than the one given on this site.
+* The [monitoring series](http://www.boxever.com/tag/monitoring) on Boxever's tech blog.
+
+## Recorded talks
+
+* [Prometheus: A Next-Generation Monitoring System](https://www.usenix.org/conference/srecon15europe/program/presentation/rabenstein) – Julius Volz and Björn Rabenstein at SREcon15 Europe, Dublin.
+* [What is your application doing right now?](youtu.be/Z0LlilNpX1U) – Matthias Gruter, Transmode, at DevOps Stockholm Meetup.
```
Comment (grobie) [inline content/docs/introduction/media.md]: Might be good to keep the scheme in the youtube URL.

File: content/docs/instrumenting/exporters.md [hunk 2]
```diff
@@ -16,37 +16,39 @@ These exporters are maintained as part of the official
...
-   * [Minecraft exporter module](https://github.com/Baughn/PrometheusIntegration)
-   * [Meteor JS web framework exporter](https://atmospherejs.com/sevki/prometheus-exporter)
    * [Memcached exporter](https://github.com/Snapbug/memcache_exporter)
+   * [Meteor JS web framework exporter](https://atmospherejs.com/sevki/prometheus-exporter)
+   * [Minecraft exporter module](https://github.com/Baughn/PrometheusIntegration)
+   * [MongoDB exporter](https://github.com/dcu/mongodb_exporter)
+   * [Munin exporter](https://github.com/pvdh/munin_exporter)
+   * [Redis exporter](https://github.com/oliver006/redis_exporter)
+   * [RethinkDB exporter](https://github.com/oliver006/rethinkdb_exporter)
+   * [scollector exporter](https://github.com/tgulacsi/prometheus_scollector)
```
Comment (grobie) [inline content/docs/instrumenting/exporters.md]: Might make sense to sort by case insensitive.

**Comment by grobie** [comment]
:+1:

**Comment by brian-brazil** [comment]
:+1:

File: content/docs/introduction/media.md (hunk 1 above)
Comment (juliusv) [inline content/docs/introduction/media.md]: :+1: for keeping the scheme - wouldn't this just be a relative link otherwise anyways?

Comment (beorn7) [inline content/docs/introduction/media.md]: Yes, obviously... copy&paste error.

File: content/docs/instrumenting/exporters.md (hunk 2 above)
Comment (beorn7) [inline content/docs/instrumenting/exporters.md]: I did that, no?

Comment (grobie) [inline content/docs/instrumenting/exporters.md]: Oops. Brainfart.

**Pull request merged**